db/crud.go

```diff
@@ -1346,7 +1346,12 @@ func (db *Database) updateAndReturnDoc(docid string, allowImport bool, expiry ui
 		}
 		db.revisionCache.Put(doc.ID, documentRevision)
 		if db.EventMgr.HasHandlerForEvent(DocumentChange) {
-			db.EventMgr.RaiseDocumentChangeEvent(body, oldBodyJSON, revChannels)
+			webhookJSON, err := doc.MarshalBodyForWebhook()
+			if err != nil {
+				base.Warnf(base.KeyAll, "Error marshalling doc for webhook post: %v", err)
```

Comment:
Maybe include some more context in the warning, like doc ID / rev id. Don't forget to redact as userdata!